tests/test_rows_update.py

```diff
@@ -0,0 +1,38 @@
+import json
+
+doc_contents = {"hello": "doc1"}
+doc_uri = "/doc1.json"
+update_query_fromDocDescriptors = (
+    'const docDescriptors = [{ uri:"'
+    + doc_uri
+    + '", doc:'
+    + json.dumps(doc_contents)
+    + "}]; op.fromDocDescriptors(docDescriptors).write()"
+)
+update_query_remove = 'op.fromDocUris("' + doc_uri + '").lockForUpdate().remove()'
+
+
+def test_update_dsl_fromDocDescriptors(admin_client):
+    response = admin_client.rows.update(
+        update_query_fromDocDescriptors, return_response=True
+    )
+    assert 200 == response.status_code
+
+    docs = admin_client.documents.read([doc_uri])
+    doc1 = next(doc for doc in docs if doc.uri == doc_uri)
+    assert "application/json" == doc1.content_type
+    assert doc1.version_id is not None
+    assert doc_contents == doc1.content
+
+
+def test_update_dsl_remove(admin_client):
```

Comment:
I think we only need a single test that performs an update - we don't care what kind of update, we just need to know that it works now. You can avoid the need for `admin_client` by instead doing the following:

1. Use `client.documents.write` to write a single doc.
2. Use an update DSL to remove that single doc.
3. Verify the doc was deleted.

That test should fail on MarkLogic 11.1 and should succeed against 11 nightly. 

I think a good second test would be to instead call `client.rows.query` with the update DSL and verify that we get an error (IIRC, Jos added an explicit error message for this scenario). Those are the kinds of tests that benefit the server team too.